proprietary/tokens/src/components/amsterdam/page-heading.tokens.json

```diff
@@ -3,15 +3,24 @@
     "page-heading": {
       "color": { "value": "{amsterdam.color.primary-black}" },
       "font-family": { "value": "{amsterdam.typography.font-family}" },
-      "line-height": { "value": "{amsterdam.typography.text-level.0.line-height}" },
-      "narrow": {
-        "font-size": { "value": "{amsterdam.typography.text-level.0.narrow.font-size}" }
-      },
-      "wide": {
-        "font-size": { "value": "{amsterdam.typography.text-level.0.wide.font-size}" }
-      },
       "font-weight": { "value": "{amsterdam.typography.font-weight.bold}" },
-      "inverse-color": { "value": "{amsterdam.color.primary-white}" }
+      "inverse-color": { "value": "{amsterdam.color.primary-white}" },
+      "density-high": {
```

Comment:
Can we decide that variation blocks like these are ordered after all general properties?